src/Asm89/Stack/CorsService.php

```diff
@@ -36,9 +36,9 @@ private function normalizeOptions(array $options = array())
             'maxAge' => 0,
         );
 
-        // normalize array('*') to true
-        if (in_array('*', $options['allowedOrigins'])) {
-            $options['allowedOrigins'] = true;
```

Comment:
I don't think this needs to be changed....

Reply:
Not technically, I just reversed the options here, so it's always an array which we can implode (and semantically closer to the actual CORS terminology). But it doesn't actually change the behavior (the array is check later on)

Reply:
I've reverted this btw.